1.0/resources/tools.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: default
+type: guide
+shortname: Tools
+title: Tools
+subtitle: tools for developing, debugging, and productizing Polymer apps
+---
+
+{% include toc.html %}
+
+The Polymer team has built modular tools for developing, debugging, and productizing Polymer apps.
+
+## Build / optimization tools {#build}
+
+### <b>vulcanize</b> - optimize HTML Imports {#vulcanize}
+
+[vulcanize](https://github.com/polymer/vulcanize) is a tool for crushing an HTML Import file and its dependent HTML Imports into one file. Think of it as a build step for web components. Vulcanize reduces the number of requests made by your application by concatenating imports into a single file.
+
+Install:
+
+    npm install -g vulcanize
+
+Usage (CLI):
+
+    vulcanize --inline-scripts --inline-css --strip-comments \
+        elements.html > elements.build.html
+
+Vulcanize can be used from the CLI or ran programmatically from Node, `gulp-vulcanize`, or `grunt-vulcanize`. Read more about vulcanize in "[Concatenating Web Components with Vulcanize](/articles/concatenating-web-components.html)".
```

Comment:
Yea. Maybe when this lands, we can create a /tools dir with more stuff in it? There could be breakout articles for everything.